Lib/bz2.py

```diff
@@ -10,9 +10,9 @@
 __author__ = "Nadeem Vawda <nadeem.vawda@gmail.com>"
 
 from builtins import open as _builtin_open
+from compression._common import streams
```

Comment:
I suggest adding an `as _streams` to this so that "streams" won't be a public name in the modules we import it within.  You could also just rename streams.py to _streams.py to avoid requiring the `as ` idioms.

Reply:
Good idea!

Reply:
Since I call this out in the PEP I will also update https://github.com/python/peps/pull/4387